src/test/ui/consts/offset_ub.stderr

```diff
@@ -150,5 +150,20 @@ LL |         intrinsics::offset(self, count)
 LL | pub const NULL_OFFSET_ZERO: *const u8 = unsafe { ptr::null::<u8>().offset(0) };
    | -------------------------------------------------------------------------------
 
-error: aborting due to 10 previous errors
+error: any use of this value will cause an error
+  --> $SRC_DIR/libcore/ptr/const_ptr.rs:LL:COL
+   |
+LL |         intrinsics::offset(self, count)
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |         |
+   |         unable to turn bytes into a pointer
+   |         inside `std::ptr::const_ptr::<impl *const u8>::offset` at $SRC_DIR/libcore/ptr/const_ptr.rs:LL:COL
+   |         inside `UNDERFLOW_ABS` at $DIR/offset_ub.rs:23:47
+   | 
+  ::: $DIR/offset_ub.rs:23:1
+   |
+LL | pub const UNDERFLOW_ABS: *const u8 = unsafe { (usize::MAX as *const u8).offset(isize::MIN) };
+   | ---------------------------------------------------------------------------------------------
```

Comment:
@oli-obk any idea why the span is so bad here? Shouldn't it point at just the function call?

Reply:
This looks to be related to the type of error that eventually bubbles up to the user. `any use of this value will cause an error` is a an error about the _value_, so that's what gets highlighted. I agree that it looks weird (though it does seem consistent with the other test cases).